backend/schemas/source/excel/libs/SheetValidations.libsonnet

```diff
@@ -86,7 +86,7 @@ local LoanBalanceValidation = {
 
 local AwardReferenceValidation = {
   type: 'custom',
-  formula1: '=AND(LEN(FIRSTCELLREF)=10, LEFT(FIRSTCELLREF, 6)="AWARD-", ISNUMBER(VALUE(MID(FIRSTCELLREF, 7, 4))), NOT(FIRSTCELLREF="AWARD-0000"))',
+  formula1: '=AND(LEN(FIRSTCELLREF)=11, LEFT(FIRSTCELLREF, 6)="AWARD-", ISNUMBER(VALUE(MID(FIRSTCELLREF, 7, 5))), NOT(FIRSTCELLREF="AWARD-00000"))',
   custom_error: 'The value should follow the pattern AWARD-#### (where #### is a four-digit number).',
```

Comment:
```suggestion
  custom_error: 'The value should follow the pattern AWARD-##### (where ##### is a five-digit number).',
```

Reply:
Given this change, we should also update the workbook version number, and the list of versions we accept. @phildominguez-gsa 

Per @jadudm the new version number should be `1.1.0`.

The workbook version number is in `excel/libs/sheets.libsonnet`. 

The list of versions we accept is in `audit/intakelib/checks/check_version_number.py`.

We should modify the `should_pass` workbooks with length five award identifiers to be version `1.1.0` to exercise the `check_version_number.py` change.

@jadudm is unsure that this is an exhaustive list of version-related changes.

Reply:
@phildominguez-gsa The output schema json should also be regenerated to include the the 1.1.0 version change - that falls under the `build_sections` portion of the Makefile

Reply:
Done, and tests pass

Reply:
Resolved through edits and reviewed by @jadudm , @phildominguez-gsa , and @tadhg-ohiggins .